Common/cpp/LayoutAnimations/LayoutAnimationsManager.cpp

```diff
@@ -159,6 +159,11 @@ int LayoutAnimationsManager::findPrecedingViewTagForTransition(const int tag) {
   return -1;
 }
 
+const std::vector<int> LayoutAnimationsManager::getSharedGroup(const int tag) {
+  auto const &groupName = viewTagToSharedTag_[tag];
```

Comment:
This is a nitpick but we mostly use `const auto &` in our codebase.
```suggestion
  const auto &groupName = viewTagToSharedTag_[tag];
```